modules/telemetry/src/specific/planetssonification.cpp

```diff
@@ -0,0 +1,779 @@
+        "Gravity",
+        "Gravity",
+        "Toggle the gravity sonification for the planet"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo TemperatureInfo = {
+        "Temperature",
+        "Temperature",
+        "Toggle the temperature sonification for the planet"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo AtmosphereInfo = {
+        "Atmosphere",
+        "Atmosphere",
+        "Toggle the atmosphere sonification for the planet"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo MoonsInfo = {
+        "Moons",
+        "Moons",
+        "Toggle the moons sonification for the planet"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo RingsInfo = {
+        "Rings",
+        "Rings",
+        "Toggle the rings sonification for the planet"
+    };
+
+    // Precision settings
+    const openspace::properties::PropertyOwner::PropertyOwnerInfo PrecisionInfo = {
+        "Precision",
+        "Precision",
+        "Settings for the distance and angle precision of the planets sonification"
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo LowDistancePrecisionInfo = {
+        "LowDistancePrecision",
+        "Distance Precision (Low)",
+        "The precision in meters used to determin when to send updated distance data "
+        "over the OSC connection. This is the lower precision used, for planets that are "
+        "not the current focus",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo HighDistancePrecisionInfo = {
+        "HighDistancePrecision",
+        "Distance Precision (High)",
+        "The precision in meters used to determin when to send updated distance data "
+        "over the OSC connection. This is the higher precision used, for the planet that "
+        "is the current focus",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo LowAnglePrecisionInfo = {
+        "LowAnglePrecision",
+        "Angle Precision (Low)",
+        "The precision in radians used to determin when to send updated angle data "
+        "over the OSC connection. This is the lower precision used, for planets that are "
+        "not the current focus",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo HighAnglePrecisionInfo = {
+        "HighAnglePrecision",
+        "Angle Precision (High)",
+        "The precision in radians used to determin when to send updated angle data "
+        "over the OSC connection. This is the higher precision used, for the planet that "
+        "is the current focus",
+        openspace::properties::Property::Visibility::User
+    };
+
+    struct [[codegen::Dictionary(PlanetsSonification)]] Parameters {
```

Comment:
This is the first type whose settings are added in a `Parameters` object.  The others won't be createable or possible to set from an asset file. Should they be?